Frontend/android-native/app/src/main/java/in/juspay/mobility/CommonJsInterface.java

```diff
@@ -1448,6 +1449,53 @@ public void firebaseUserID(String id) {
         mFirebaseAnalytics.setUserId(id);
     }
 
+    CleverTapAPI clevertapDefaultInstance = CleverTapAPI.getDefaultInstance(context);
+    boolean isCleverTapInstanceCreated = true;
+
+    @JavascriptInterface
+    public void setCleverTapUserData(String key, String value){
+        HashMap<String, Object> profileUpdate = new HashMap<String, Object>();
+        try {
+            System.out.println("it's entered data"+key+value);
```

Comment:
@ashkriti  can you remove all the spy statements.

Reply:
At the moment I have added the spy statements to check and it was suggested by @witcher-shailesh